eox-core-v-1-0/prose/edit/src/design-considerations-00.md

```diff
@@ -1,13 +1,69 @@
 # Design Considerations
 
-OpenEoX is a language to exchange any End-of information formulated in JSON.
+The OpenEoX framework is a comprehensive language to exchange End-of-X (EoX) lifecycle information formulated in JSON, addressing the critical industry need for standardized communication of product lifecycle stages across commercial and open-source software and hardware.
 
-The standard is divided into 3 parts:
+## Framework Architecture
 
-- OpenEoX Core (this specification): contain the OpenEoX core information and intended to be imported by other standards which will provide the
-  product context needed.
-  Optionally, the product context can be omitted if the product itself provides the information, e.g. through in an SNMP or HTTPS response.
-- OpenEoX Shell: binding OpenEoX core information to products.
-  Such combination is called an OpenEoX statement.
-  The OpenEoX Shell forms the so called standalone mode in OpenEoX.
-- OpenEoX API: deals with the distribution of OpenEoX Core and OpenEoX Shell data.
+The OpenEoX standard is architected as a modular three-layer framework:
+
+### OpenEoX Core (this specification)
+Contains the fundamental EoX lifecycle information elements and is designed to be imported by other standards or integrated directly into product responses. The core schema establishes the essential data elements required for any EoX statement:
+
+- **end_of_life**: The definitive timestamp when all vendor support ceases
+- **end_of_security_support**: The critical timestamp when security remediation commitments end
+- **end_of_sales**: The commercial availability termination timestamp (optional)
+- **last_updated**: RFC 3339 timestamp tracking for data freshness and change management
+
+The core information can be provided independently when the product context is available through other means, such as SNMP responses, HTTPS API responses.
+
+### OpenEoX Shell
+Provides the binding mechanism that combines OpenEoX core lifecycle information to specific product identifications. This combination is called an "OpenEoX statement" - a complete, standalone representation of lifecycle information for a specific product.
+
+The Shell schema supports multiple product types through specialized identification schemas:
+- **Software Products**: Identified by vendor name, product name, and version
+- **Hardware Products**: Identified by vendor name, product name, model, and serial number ranges
+- **Hardware with Software Products**: Combined identification supporting both hardware and embedded software components
```

Comment:
As this is just a generic overview, I would skip this part here (to avoid duplication / contradiction with the OpenEoX Shell specification).